RFunction.R

```diff
@@ -7,10 +7,11 @@ rFunction <- function(data)
   
   data.spdf <- SpatialPointsDataFrame(coordinates(data),as.data.frame(data), proj4string=CRS("+proj=longlat +ellps=WGS84 +no_defs"))
   
-  targetDirZipFile <- Sys.getenv(x = "APP_ARTIFACTS_DIR", "/tmp")
+  dir.create(targetDirZipFile <- Sys.getenv(x = "APP_ARTIFACTS_DIR", "artefact-mock"))
```

Comment:
This is a sample how to work with the special `APP_ARTEFACTS_DIR` in the local r-studio:

- the last param (`artefact-mock`) is for the developer. This directory will be created on the machine of the app-developer
- the environment var `APP_ARTIFACTS_DIR` will overwrite this on MoveApps.